settings.cpp

```diff
@@ -93,8 +93,12 @@ void PTZSettings::RefreshLists()
 	}
 
 	ui->viscaPortComboBox->clear();
-	Q_FOREACH(QSerialPortInfo port, QSerialPortInfo::availablePorts())
+	ui->pelcoPComboBox->clear();
+	Q_FOREACH(QSerialPortInfo port, QSerialPortInfo::availablePorts()) {
 		ui->viscaPortComboBox->addItem(port.portName());
+		ui->pelcoPComboBox->addItem(port.portName());
+	}
+		
```

Comment:
I think the viscaPort and viscaAddress fields can be reused for PELCO connections too. So, instead of adding additional fields, change the name on the label to "VISCA Serial" or "PELCO-P Serial" depending on what kind of camera is selected. That will remove the need to double this code up.